casper/src/main/resources/AuthKey.rho

```diff
@@ -74,11 +72,9 @@ in {
   contract @(*AuthKey, "extractState")(ret) = {
     ret!({"authKeyUnf": *_authKey})
   } |
-  // Insert location channel into the registry location map
-  rb!(`rho:rchain:authKey`, *AuthKey, *AuthKeyDispatcher) |
   // Inserts signed write-only AuthKey contract into the registry
   rs!(
-    (9223372036854775807, bundle+{*AuthKeyDispatcher}),
+    (9223372036854775807, *AuthKey),
```

Comment:
Why is it not bundled anymore?

Reply:
https://github.com/rchain/rchain/pull/3485#discussion_r866737455

Reply:
requested from @tgrospic

Reply:
The `AuthKey` is now hidden behind `Dispatcher` which users no longer have access.

Reply:
This now implies some implicit management for updatable contract. I had something like this in mind where insert to the registry is the same for all. Updatable contract should be created explicitly.
```scala
proxify!(*RevVault, *updateSecurityLogicOrSomething, *RevVaultProxyCh) |
for(RevVaultProxy <- RevVaultProxyCh) {
  rs!(
    (9223372036854775807, bundle+{*RevVaultProxy}),
    *deployerId,
    *uriOut)
}
```